lms/templates/register.html

```diff
@@ -16,6 +16,15 @@
 <%! from third_party_auth import pipeline, provider %>
 <%! import calendar %>
 
+<%
+  if self.theme_enabled() and not self.is_microsite():
```

Comment:
Actually, let's take that thought even further...

Could we move all of these `*_extra_file = ...` calls into a single location? 

So somewhere, maybe `base.html` or some such, we'd have:

```python
register_extra_file = None
honor_link = None
register_sidebar_file = None
if self.is_non_microsite_theme():
    register_extra_file = 'theme-register_extra.html
    honor_link = marketing_link('TOS') + '#honor'
    register_sidebar_file = 'theme-register_sidebar.html'
    # etc...
```

This approach would save a _lot_ of duplicate code.